2021-08-26 06:41:14,613 INFO    MainThread:17359 [wandb_setup.py:_flush():69] setting env: {}
2021-08-26 06:41:14,613 INFO    MainThread:17359 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-26 06:41:14,613 INFO    MainThread:17359 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '84f10b11a8fa0d0036151625c778049a02f3728b'}
2021-08-26 06:41:14,613 INFO    MainThread:17359 [wandb_init.py:_log_setup():342] Logging user logs to /opt/ml/image-classification-level1-25/wandb/run-20210826_064114-3am5kidr/logs/debug.log
2021-08-26 06:41:14,613 INFO    MainThread:17359 [wandb_init.py:_log_setup():343] Logging internal logs to /opt/ml/image-classification-level1-25/wandb/run-20210826_064114-3am5kidr/logs/debug-internal.log
2021-08-26 06:41:14,614 INFO    MainThread:17359 [wandb_init.py:_jupyter_setup():294] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7f6bc1e39760>
2021-08-26 06:41:14,614 INFO    MainThread:17359 [wandb_init.py:init():375] calling init triggers
2021-08-26 06:41:14,614 INFO    MainThread:17359 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {}
2021-08-26 06:41:14,614 INFO    MainThread:17359 [wandb_init.py:init():424] starting backend
2021-08-26 06:41:14,614 INFO    MainThread:17359 [backend.py:_multiprocessing_setup():70] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2021-08-26 06:41:14,647 INFO    MainThread:17359 [backend.py:ensure_launched():135] starting backend process...
2021-08-26 06:41:14,673 INFO    MainThread:17359 [backend.py:ensure_launched():139] started backend process with pid: 17547
2021-08-26 06:41:14,674 INFO    MainThread:17359 [wandb_init.py:init():429] backend started and connected
2021-08-26 06:41:14,694 INFO    MainThread:17359 [wandb_run.py:_label_probe_notebook():798] probe notebook
2021-08-26 06:41:14,694 INFO    MainThread:17359 [wandb_run.py:_label_probe_notebook():805] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2021-08-26 06:41:14,695 INFO    MainThread:17359 [wandb_init.py:init():477] updated telemetry
2021-08-26 06:41:14,695 INFO    MainThread:17359 [wandb_init.py:init():500] communicating current version
2021-08-26 06:41:15,508 INFO    MainThread:17359 [wandb_init.py:init():505] got version response 
2021-08-26 06:41:15,508 INFO    MainThread:17359 [wandb_init.py:init():513] communicating run to backend with 30 second timeout
2021-08-26 06:41:15,725 INFO    MainThread:17359 [wandb_init.py:init():540] starting run threads in backend
2021-08-26 06:41:20,274 INFO    MainThread:17359 [wandb_run.py:_console_start():1584] atexit reg
2021-08-26 06:41:20,275 INFO    MainThread:17359 [wandb_run.py:_redirect():1458] redirect: SettingsConsole.WRAP
2021-08-26 06:41:20,277 INFO    MainThread:17359 [wandb_run.py:_redirect():1495] Wrapping output streams.
2021-08-26 06:41:20,280 INFO    MainThread:17359 [wandb_run.py:_redirect():1519] Redirects installed.
2021-08-26 06:41:20,280 INFO    MainThread:17359 [wandb_init.py:init():565] run started, returning control to user process
2021-08-26 06:41:20,281 INFO    MainThread:17359 [wandb_init.py:_pause_backend():256] pausing backend
2021-08-26 06:41:20,281 INFO    MainThread:17359 [jupyter.py:save_ipynb():284] not saving jupyter notebook
2021-08-26 06:41:24,902 INFO    MainThread:17359 [wandb_init.py:_resume_backend():265] resuming backend
2021-08-26 06:41:24,912 INFO    MainThread:17359 [wandb_setup.py:_flush():69] setting env: {}
2021-08-26 06:41:24,913 INFO    MainThread:17359 [wandb_setup.py:_flush():69] setting login settings: {}
2021-08-26 06:41:24,913 INFO    MainThread:17359 [wandb_setup.py:_flush():69] setting login settings: {'api_key': '84f10b11a8fa0d0036151625c778049a02f3728b'}
2021-08-26 06:41:24,913 INFO    MainThread:17359 [wandb_init.py:_log_setup():342] Logging user logs to /opt/ml/image-classification-level1-25/wandb/run-20210826_064124-zznditlp/logs/debug.log
2021-08-26 06:41:24,913 INFO    MainThread:17359 [wandb_init.py:_log_setup():343] Logging internal logs to /opt/ml/image-classification-level1-25/wandb/run-20210826_064124-zznditlp/logs/debug-internal.log
2021-08-26 06:41:24,913 INFO    MainThread:17359 [wandb_init.py:init():375] calling init triggers
2021-08-26 06:41:24,913 INFO    MainThread:17359 [wandb_init.py:init():380] wandb.init called with sweep_config: {}
config: {'epochs': 3, 'batch_size': 32, 'learning_rate': 0.001}
2021-08-26 06:41:24,914 INFO    MainThread:17359 [wandb_init.py:init():395] re-initializing run, found existing run on stack: 3am5kidr
2021-08-26 06:41:24,915 INFO    MainThread:17359 [wandb_run.py:finish():1254] finishing run presto105/lv1_p/3am5kidr
